Helper/OmiseHelper.php

```diff
@@ -39,4 +41,69 @@ public function omiseAmountFormat($currency, $amount)
 
         return $amount;
     }
+
+    /**
+     * Convert tesco code returned from Omise Backend in SVG format to HTML format
+     * 
+     * @param  string  $svg
+     *
+     * @return string
+     */
+    public function convertTescoSVGCodeToHTML($svg)
```

Comment:
@jacstn I guess this method can be named a bit more in generic way? 
Maybe `convertSVGToHtml`? 

Or maybe can even have a class `Helper\BarcodeHelper.php`?

Reply:
@jacstn then do we really need this method to be in `OmiseHelper`?
How about just have it as a private or protected method in the `TescoPaymentObserver.php` class as it will be used only in that place?

Reply:
well.. could be the case.. but
I will add in the future generation of PDF document with that code, like there is in AirAsia.

I am thinking that maybe good idea would be to convert it as soon as it arrives in response. But than again.. adding that function to response is not good idea.
Than it would be also displayed on store page as html, now it is SVG.

Reply:
BTW. that pdf would be available in checkout success page.